documentation/controls.txt

```diff
@@ -0,0 +1,18 @@
+
+Driver:
```

Comment:
From Solomon: Driver should have a function on the B button which swaps the forward and rotate axes. For example, Right X becomes Left X for rotate and Left Y becomes Right Y for forward. We can default to whatever we prefer.

Reply:
We should implement this one before merging. Unsure if we should use the B button since that's currently part of the safety, although we could move the safety to other buttons.

Reply:
we don't want to do this